resources/delete_from_list.rb

```diff
@@ -2,15 +2,16 @@
 property :pattern, [String, Regexp]
 property :delim, Array
 property :entry, String
-property :ignore_missing, [true, false], default: false
 property :eol, String, default: Line::OS.unix? ? "\n" : "\r\n"
 property :backup, [true, false], default: false
+property :ignore_missing, [true, false], default: true
 
 resource_name :delete_from_list
 
 action :edit do
-  return if !::File.exist?(new_resource.path) && new_resource.ignore_missing
-  raise "File #{new_resource.path} not found" unless ::File.exist?(new_resource.path)
+  file_exist = ::File.exist?(new_resource.path)
```

Comment:
what do you think about moving file_exist(s) into the helpers and removing that duplication?

Reply:
Sure